test/plugin/scenarios/webflux-scenario/webflux-projectA-scenario/src/main/java/org/apache/skywalking/apm/testcase/sc/webflux/projectA/controller/TestController.java

```diff
@@ -43,6 +45,7 @@ public String testcase() throws IOException {
         visit("http://" + hostBAddress + "/testcase/route/error");
         visit("http://" + hostBAddress + "/notFound");
         visit("http://" + hostBAddress + "/testcase/annotation/mono/hello");
+        visit("http://" + clientHostAddress + "/testcase/webclient/testGet");
```

Comment:
What is the difference between this new link and above L45-L47? You seem only to change the target address. Why you do `WebClient` in another new service rather than doing directly here? I am feeling the new service in the test is not necessary.

Please correct me if I am wrong.

Reply:
Sir,The reason why I do `WebClient` in another new  is that I fear it will appear thread confusion .

Reply:
I mean, you should do `WebClient` at L48 directly(of course the new method is fine). I just don't think you should do RPCs twice.

Reply:
You mean I should use `WebClient` to creat a request rather than `visit()`,should I?

Reply:
Yes. This RPC is not necessary

Reply:
Ok,I will modify it.